Mathlib/LinearAlgebra/TensorProductBasis.lean

```diff
@@ -54,5 +54,12 @@ theorem Basis.tensorProduct_apply' (b : Basis ι R M) (c : Basis κ R N) (i : ι
     Basis.tensorProduct b c i = b i.1 ⊗ₜ c i.2 := by simp [Basis.tensorProduct]
 #align basis.tensor_product_apply' Basis.tensorProduct_apply'
 
-end CommRing
+-- Porting note: resolved diamond
+set_option synthInstance.etaExperiment true in
+@[simp]
+theorem Basis.tensorProduct_repr_tmul_apply (b : Basis ι R M) (c : Basis κ R N) (m : M) (n : N)
+    (i : ι) (j : κ) : (Basis.tensorProduct b c).repr (m ⊗ₜ n) (i, j) = b.repr m i * c.repr n j := by
```

Comment:
To make it easier to quickly spot the conclusion of the lemma:
```suggestion
    (i : ι) (j : κ) :
    (Basis.tensorProduct b c).repr (m ⊗ₜ n) (i, j) = b.repr m i * c.repr n j := by
```

Reply:
This is just the mathport output; it would be nice if it had this heuristic internall (cc @digama0)

Either way, happy to make the change.